web/src/compose_recipient.ts

```diff
@@ -394,14 +396,15 @@ export function update_topic_displayed_text(topic_name = "", has_topic_focus = f
     compose_state.topic(topic_name);
 
     const $input = $("input#stream_message_recipient_topic");
-    const is_empty_string_topic = topic_name === "";
+    $input.prop("disabled", false);
+    $("#compose_recipient_box").removeClass("disabled");
```

Comment:
Should this by part of the block with the `// reset` comment? I think it's also reset code, right?